.env.test

```diff
@@ -1,6 +1,7 @@
 # define your env variables for the test env here
 KERNEL_CLASS='App\Kernel'
 APP_SECRET='$ecretf0rt3st'
+APP_ENV=test
```

Comment:
This is usually not needed. .env.test is already only loaded when the APP_ENV is test.